Views/Widget-OrderedList.liquid

```diff
@@ -0,0 +1,10 @@
+{% assign id = Model.ContentItem.Content.HtmlAttributesPart.Id %}
+{% assign cssClasses = Model.ContentItem.Content.HtmlAttributesPart.CssClasses %}
+
+{% if Model.ContentItem.Content.HtmlAttributesPart != null %}
```

Comment:
Ha, only reason I added the `if` statement on `HtmlAttributePart` existing was I didn't like the whitespace that was being rendered in the output. I could squash the inline `if`s but thought that would look super ugly.

Reply:
Ah ok, I was also looking at those `if`s and thinking they should be squashed to prevent whitespace, I think sometimes these things are necessary when we're working with a templating language! haha.